src/components/scenes/newAccount/NewAccountUsernameScene.tsx

```diff
@@ -43,6 +41,13 @@ export const NewAccountUsernameScene = ({ branding }: Props) => {
     false
   )
 
+  const initUsername = useSelector(state => state.create.username ?? '')
+
+  const mounted = React.useRef<boolean>(true)
+  const fetchCounter = React.useRef<number>(0)
+
+  React.useEffect(() => setUsername(initUsername), [initUsername])
```

Comment:
We should not use an effect to initialize state. The `useState` hook takes a value, so just provide that:
`const [username, setUsername] = React.useState(initUsername ?? '')`. We don't want the username changing out from under the user while they are typing.